### PR TITLE
Correct formatting for TaskRun in Build docs

### PR DESCRIPTION
**Pull request by likhithj1996**
Version(s):

builds-docs-1.1` and `builds-docs-1.0

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6154

Link to docs preview:

[BuildRun Resource](https://85708--ocpdocs-pr.netlify.app/openshift-builds/latest/about/overview-openshift-builds.html#about-buildrun_overview-openshift-builds)

QE review:
- [ ] QE has approved this change.

QE review:
SME review:
Peer review:
Internal Peer review: @Dhruv-Soni11 

Additional information:

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
/lgtm, thanks @likhithj1996. 

Kindly add the correct hyperlink of the affected section in the PR description and do not copy-paste the automated preview link. The change was made in the **buildrun resource** section but you have pasted a link for the **Overview of builds** section, which is incorrect.

**Comment by likhithj1996** [comment]
@Dhruv-Soni11 Thanks Dhruv. I have updated the PR with the correct preview link.

**Comment by shipsing** [comment]
@likhithj1996, for clarity purpose: It is not always necessary to put TaskRun or BuildRun in backticks. The cases where we want to refer these entities in generic manner, we can simply write task run or build run. You need to use backticks only when you refer those entities as objects or resources. for example, A `BuildRun` resource invokes a build on your cluster, similar to any cluster job or Tekton task run. In the sentence, BuildRun needs backticks because we are referring it as a resource, but task run is used in generic manner (no need of backticks).

You can refer to [this link](https://docs.openshift.com/container-platform/4.8/cicd/pipelines/understanding-openshift-pipelines.html) to identify pipeline run instances that do not use backticks.

I do not think we should put task run in backticks in this context.

**Comment by shipsing** [comment]
@likhithj1996, I would suggest to close this PR and the [issue](https://issues.redhat.com/browse/RHDEVDOCS-6154). There is no need to change anything in the task run formatting.

File: modules/ob-about-buildrun.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="about-buildrun_{context}"]
 = BuildRun resource
 
-A `BuildRun` resource invokes a build on your cluster, similar to any cluster job or Tekton task run. The `BuildRun` resource represents a workload on your cluster, which results in a running pod. A `BuildRun` is the running instance of a build. It instantiates a build for execution with specific parameters on a cluster.
```
Comment (shipsing) [inline modules/ob-about-buildrun.adoc:9]: @likhithj1996, here the TaskRun is referred in a generic way, that is, task run. So, no need to change its formatting. Please close the PR, the original formatting here is correct.

Comment (likhithj1996) [reply modules/ob-about-buildrun.adoc:9]: Thanks for the explanation in simple and clear way. I agree and will be closing the PR and the JIRA ticket. Thank you.

**Pull request closed without merging**